dist/package.sh

```diff
@@ -78,12 +78,23 @@ targets=(
     sdp/sdp.js
 )
 
+
 for target in "${targets[@]}"; do
-    install -D "node_modules/$target" "$DIR/node_modules/$target"
+
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		ditto "node_modules/$target" "$DIR/node_modules/$target"
+    else
+		install -D "node_modules/$target" "$DIR/node_modules/$target"
+    fi
```

Comment:
You have a mixture of spaces and tabs in your files. Can you fix that? :) We use 4 spaces to indent.

Reply:
Fixed in https://github.com/threema-ch/threema-web/pull/406/commits/06bca90125ccc1e6e5c309c07c2c61c85cd46449